### PR TITLE
Update titanium types to 8.0.0

### PR DESCRIPTION
**Pull request by ewanharris**
Please fill in this template.

- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [ ] Add or edit tests to reflect the change. (Run with `npm test`.)
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [x] Run `npm run lint package-name` (or `tsc` if no `tslint.json` is present).

Select one of these and delete the others:

If changing an existing definition:
- [x] Provide a URL to documentation or source code which provides context for the suggested changes: https://devblog.axway.com/mobile-apps/ga-release-titanium-sdk-8-0-0-appcelerator-cli-7-0-10-appcelerator-studio-5-1-2/
- [x] Increase the version number in the header if appropriate.
- [ ] If you are making substantial changes, consider adding a `tslint.json` containing `{ "extends": "dtslint/dt.json" }`.

### CODE REVIEW

File: types/titanium/index.d.ts [hunk 1]
```diff
@@ -16880,11 +18252,6 @@ declare namespace Titanium {
 			 */
 			getHtml(): string;
 
-			/**
-			 * Sets the value of the <Titanium.UI.WebView.html> property.
-			 */
-			setHtml(html: string): void;
```
Comment (janvennemann) [inline types/titanium/index.d.ts]: This shouldn't be removed. See https://github.com/appcelerator/titanium_mobile/blob/master/apidoc/Titanium/UI/WebView.yml#L145, this is not just a simple setter. Not sure if this is an issue with the TS definition generator or the docs parser, though.

**Review by janvennemann — CHANGES_REQUESTED** [review]

**Review by janvennemann — CHANGES_REQUESTED** [review]
See suggested changes above.

File: types/titanium/index.d.ts (hunk 1 above)
Comment (ewanharris) [reply types/titanium/index.d.ts]: I think this one was a dupe of the one with both the html and options parameters on [line 18041](https://github.com/DefinitelyTyped/DefinitelyTyped/pull/34512/files#diff-57636c23872b22c51a9183be5b85f9f9R18041). For methods like this that matched a setter but also had a declared method the doc system would dupe them, [this commit](https://github.com/appcelerator/titanium_mobile/commit/b5d6124918bf1bb12981ffaf6b2cb7c46741b2b5) fixed that

Comment (janvennemann) [reply types/titanium/index.d.ts]: Ah, perfect!

**Review by janvennemann — APPROVED** [review]
Looking good!

**Comment by uniqueiniquity** [comment]
Congratulations on your first DefinitelyTyped contribution!
Thank you for being a part of the community!

**Pull request merged**